.travis.yml

```diff
@@ -32,6 +32,7 @@ install:
     pip install --upgrade setuptools
     pip install $NUMPY
     pip install $MPL
+    pip install -r requirements.txt
   - |
     cd geos-3.3.3
     export GEOS_DIR=$HOME/.local/
```

Comment:
This is failing because I removed the code from setup.py to compile _geoslib.c which is for GEOS library.  Oops.

At first glance, [shapely](http://toblerity.org/shapely/manual.html) looks like it could be replaced GEOS.  Also, that would put the burden of installing GEOS onto shapely package.